canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -101,6 +107,32 @@ $toolbar-divider-width: 1px;
 	}
 }
 
+.toolbar-div.toolbar-sm-div .toolbar-item {
```

Comment:
These styles can be nested inside the the  `&.toolbar-sm-div {`  rule set above.

Reply:
Updated it.